src/main/java/org/isf/patient/service/PatientIoOperations.java

```diff
@@ -133,11 +135,13 @@ public Patient getPatient(String name) throws OHServiceException {
 	 * @return the Patient
 	 * @throws OHServiceException
 	 */
-	public Patient getPatient(Integer code) throws OHServiceException {
```

Comment:
would it make sense to leave this method signature and make it look like:
<pre>
public Patient getPatient(Integer code) throws OHServiceException {
     return this(code, true);
}
</pre>

It seems like loading from the DB is the default and this would make it unnecessary to modify all those calls with a `true` parameter.